internal/cli/iam/projects/create.go

```diff
@@ -78,57 +87,85 @@ func (opts *CreateOpts) newCreateProjectOptions() *atlas.CreateProjectOptions {
 }
 
 func (opts *CreateOpts) validateOwnerID() error {
-	if opts.projectOwnerID == "" {
+	if opts.projectOwnerID == "" || opts.serviceVersion == nil {
 		return nil
 	}
 
-	if config.Service() != config.OpsManagerService {
+	constrain, err := semver.NewConstraint(">= 6.0")
+	if err != nil {
+		return err
+	}
+
+	if !constrain.Check(opts.serviceVersion) {
+		return fmt.Errorf("%s is available only for Atlas, Cloud Manager and Ops Manager >= 6.0", flag.OwnerID)
+	}
+
+	return nil
+}
+
+func (opts *CreateOpts) validateWithoutDefaultAlertSettings() error {
+	if !opts.withoutDefaultAlertSettings || opts.serviceVersion == nil {
 		return nil
 	}
 
-	v, err := opts.store.ServiceVersion()
+	constrain, err := semver.NewConstraint(">= 6.0")
 	if err != nil {
 		return err
 	}
 
-	sv, err := cli.ParseServiceVersion(v)
+	if !constrain.Check(opts.serviceVersion) {
+		return fmt.Errorf("%s is available only for Atlas, Cloud Manager and Ops Manager >= 6.0", flag.WithoutDefaultAlertSettings)
+	}
+
+	return nil
+}
+
+func (opts *CreateOpts) newServiceVersion() error {
```

Comment:
[nit] not a new instance 
```suggestion
func (opts *CreateOpts) initServiceVersion() error {
```

Reply:
updated